.github/workflows/lib-tests.yaml

```diff
@@ -58,6 +58,8 @@ jobs:
         env:
           test_dir: /github/workspace/libs/${{ matrix.library }}
         run: |
+            apt-get update
```

Comment:
@wbenoit26 I think doing `sudo apt-get update` (and adding `sudo` to the system install commands) might help your problem

Reply:
I found the problem after trying all of this yesterday - it's actually that GitHub CLI's GPG key [expired](https://github.com/cli/cli/issues/9569) on September 6th, so running `update` fails to find the correct signature.

The container we use automatically uses the most recent GPG key, so this should be fixed by re-building the image. I filed an issue on the `pyaction` repo yesterday, and the owner said he'd take a look at it today.

Reply:
okay cool. Is this a result of having to install `gcc` for pycbc?

Reply:
Yeah, entirely. I wouldn't need any of the `apt-get` steps if it weren't for that, which is a little annoying.